src/FondOfSpryker/Glue/ProductListsRestApi/ProductListsRestApiDependencyProvider.php

```diff
@@ -50,12 +53,28 @@ protected function addProductListCustomerClient(Container $container): Container
      */
     protected function addProductListCompanyClient(Container $container): Container
     {
-        $container[static::CLIENT_PRODUCT_LIST_COMPANY] = function (Container $container) {
+        $container[static::CLIENT_PRODUCT_LIST_COMPANY] = static function (Container $container) {
```

Comment:
why do you user static?

Reply:
because it is faster. @gengjozsef install https://plugins.jetbrains.com/plugin/7622-php-inspections-ea-extended-. this tool shows a warning for none static inline functions.